cmd/mqtt/main.go

```diff
@@ -122,6 +127,37 @@ func main() {
 		log.Fatalf(err.Error())
 	}
 
+	if cfg.mqttTargetHealthCheck != "" {
+
+		// An operation that may fail.
+		operation := func() error {
+			res, err := http.Get(cfg.mqttTargetHealthCheck)
```

Comment:
Consider extracting this function outside of main and just use it in `backoff.RetryNotify` call.